ros2topic/ros2topic/verb/pub.py

```diff
@@ -83,6 +83,9 @@ def add_arguments(self, parser, cli_name):
         group.add_argument(
             '-t', '--times', type=nonnegative_int, default=0,
             help='Publish this number of times and then exit')
+        parser.add_argument(
+            '-m', '--wait-matching-subscriptions', type=nonnegative_int, default=1,
```

Comment:
This is very subjective, but `-w` feels more natural to me as the short version of the flag.  What do others think?

Reply:
Sounds fair to me.

I think I picked `-m` because I also was going to rename the long flag to `--matching-subscriptions`, but I ended up mixing both.
I don't have a strong opinion, either is fine to me.

Reply:
Fixed in https://github.com/ros2/ros2cli/pull/642/commits/bf92afecf75376ff2d08c1b39e802d875dafb93c.